rpc/block_svc_routines.c

```diff
@@ -3956,8 +3956,8 @@ blockValidateCommandOutput(const char *out, int opt, void *data)
     /* target iqn create validation */
     GB_OUT_VALIDATE_OR_GOTO(out, out, "target iqn creation failed for: %s",
                     cblk, cblk->volume,
-                    "Created target iqn.2016-12.org.gluster-block:%s.",
-                    cblk->gbid);
+                    "Created target  %s:%s.",
```

Comment:
Very minor correction.
Can you please merge line 3959 at 3958

Reply:
Done. @pkalever  PTAL